projects/1.20/assets/mffs/mffs/lang/zh_cn.json

```
@@ -0,0 +1,194 @@
+  "info.mffs.field_permission.use_blocks": "使用方块",
+  "info.mffs.field_permission.use_blocks.tooltip": "允许玩家使用方块或打开方块GUI",
+  "info.mffs.field_permission.place_blocks": "放置方块",
+  "info.mffs.field_permission.place_blocks.tooltip": "允许玩家放置或破坏方块",
+  "info.mffs.field_permission.configure_security_center": "配置安全性设置",
+  "info.mffs.field_permission.configure_security_center.tooltip": "允许玩家配置生物身份识别器",
+  "info.mffs.field_permission.bypass_defense": "通过防线",
+  "info.mffs.field_permission.bypass_defense.tooltip": "允许玩家越过拒止矩阵的限制",
+  "info.mffs.field_permission.bypass_confiscation": "免除搜身",
+  "info.mffs.field_permission.bypass_confiscation.tooltip": "允许玩家越过拒止矩阵的违禁物品检测",
+  "info.mffs.field_permission.remote_control": "遥控",
+  "info.mffs.field_permission.remote_control.tooltip": "允许玩家在力场范围内使用遥控",
+  "info.mffs.interdiction_matrix.warning": "[%s] 警告! 您已进入监控范围，前方为限制区域!",
+  "info.mffs.interdiction_matrix.no_entry": "[%s] ，您没有进入权限，请立即离开此区域!",
+  "info.mffs.interdiction_matrix.no_permission": "[%s] 你没有权限做这个!",
+  "info.mffs.interdiction_matrix.fairwell": "[%s] 被抬走了",
+  "info.mffs.interdiction_matrix.confiscation_singular": "[%s] 你的 %s 被没收了",
+  "info.mffs.interdiction_matrix.confiscation_plural": "[%s] 你的 %s 被没收了",
+  "info.mffs.show_details": "按住 [%s] 查看详细描述",
+  "info.mffs.key.shift": "Shift",
+  "info.mffs.fortron_usage": "Fortron: %s L/s",
+  "info.mffs.handbook.title": "力场防御手册", 
+  "info.mffs.handbook.landing_text": "欢迎使用MFFS用户手册，它将引领你进入力场科技的奇妙世界。从力场到高科技的机器，再到多种多样的防御手段，这里有你想知道的一切！让我们从这里开始吧！", 
+  
+  "death.attack.mffs.field_shock": "%1$s 被力场分解了",
+  
+  "block.mffs.machine_block": "机器外壳",
+  "block.mffs.force_field": "防御力场",
+  "block.mffs.projector": "力场投射器",
+  "block.mffs.coercion_deriver": "能量转换器",
+  "block.mffs.fortron_capacitor": "Fortron电容器",
+  "block.mffs.biometric_identifier": "生物身份识别器",
+  "block.mffs.interdiction_matrix": "拒止矩阵",
+  
+  "item.mffs.remote_controller": "遥控器",
+  "item.mffs.remote_controller.description": "可以远程控制机器 Shift+右键绑定机器",
+  "item.mffs.cube_mode": "形状[立方体]",
+  "item.mffs.sphere_mode": "形状[球体]",
+  "item.mffs.tube_mode": "形状[管状]",
+  "item.mffs.pyramid_mode": "形状[金字塔]",
+  "item.mffs.cylinder_mode": "形状[圆柱体]",
+  "item.mffs.custom_mode": "形状[自定义]",
+  "item.mffs.custom_mode.mode": "模式: %s",
+  "item.mffs.custom_mode.mode.additive": "相加",
+  "item.mffs.custom_mode.mode.subtractive": "相减",
+  "item.mffs.custom_mode.primary_point": "第一个位置: %s",
+  "item.mffs.custom_mode.secondary_point": "第二个位置: %s",
+  "item.mffs.custom_mode.pattern_id": "图形ID: %s",
+  "item.mffs.custom_mode.field_size": "力场大小: %s",
+  "item.mffs.custom_mode.not_saved": "力场未保存",
+  "item.mffs.custom_mode.set_primary_point": "选择第二个位置",
+  "item.mffs.custom_mode.set_secondary_point": "[Shift+右键保存]",
+  "item.mffs.custom_mode.data_saved": "力场已保存",
+  "item.mffs.custom_mode.changed_mode": "选择模式: %s",
+  "item.mffs.custom_mode.clear": "力场已清除",
+  "item.mffs.custom_mode.too_far": "选择范围过大! 最多只能距离%s方块远",

```

```suggestion
  "item.mffs.custom_mode.too_far": "选择范围过大！最多只能距离%s方块远",
```
[要求] _如无排版需要等特殊要求，请将翻译文本中的**标点**（不包括`%3$s` `§r`等标记符和JSON格式的半角引号等）全部改为**全角**标点。_

建议全文通查

---

已改正，感谢指出